dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -1050,6 +1053,13 @@ public ResponseEntity<RepresentationModel<?>> delete(HttpServletRequest request,
         return deleteInternal(apiCategory, model, uuid);
     }
 
+    @RequestMapping(method = RequestMethod.DELETE, consumes = {"text/uri-list"})
```

Comment:
@JensRoets-Atmire and @benbosman :  Unfortunately, I don't this this is valid.  A `DELETE` request cannot contain a body/data.

See the discussion about this in this StackOverflow: https://stackoverflow.com/questions/299628/is-an-entity-body-allowed-for-an-http-delete-request

The general consensus seems to be that one should *never* send a body in a `DELETE` request because, while the specs don't disallow it, some proxies may not be able to process it, and some clients may not be able to send it.  So, going down this path can cause issues in the long run.

Looking around to find a better way to achieve this, the most frequent idea appears to be to pass multiple ids/uris as *parameters* to the request.  So, possibly we could restructure this to be more like: `DELETE /bitstreams?id=[first-uuid]&id=[second-uuid]`.

The only other approach I've seen mentioned is that some APIs like AWS use a `POST` to send a batch of requests, including batch deletions.  Here's more on that concept https://stackoverflow.com/a/41539479/3750035

Pulling in @abollini into this discussion as well, as this seems to be a concept we have not yet tackled in our REST API & may require some discussion among us all.  I don't believe we support batch deletion, or any batch actions (other than PATCH which sometimes can batch several edits at once).